Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM adoptopenjdk/openjdk11:alpine-jre
+WORKDIR /opt/app
```

Comment:
Choosing root of file system will be more simpler.

Reply:
It is a good practice to put your jar file in a folder. And generaly the folder used is _/opt/app_.  Check the [adoptopenjdk ](https://hub.docker.com/r/adoptopenjdk/openjdk8) page.
I would suggest you to keep it like this.

Reply:
I know, so no need to change just make sure everything is working.